roles/rook/tasks/rook-jenkins-update.yml

```diff
@@ -0,0 +1,38 @@
+---
+  - name: Gather facts
+    ec2_instance_facts:
+      filters:
+        "tag:Name": "{{ master_name }}"
+        instance-state-name: running
+    register: master_metadata
+
+  - name: create date
+    shell: "date +%d-%m-%y"
```

Comment:
Is there a particular reason you're getting the date and time from the host instead of using ansible's built-in time function?  https://stackoverflow.com/a/31323923/6225562

Reply:
Fixed it